### PR TITLE
WEBEU2 - Nmeregini Vincent -Giacomo Benati- Week 3 Day 1

### PR DESCRIPTION
**Pull request by iamcristos**

### CODE REVIEW

File: assignments/objects.js [hunk 1]
```diff
@@ -18,28 +18,68 @@ const example = {
...
+console.log(antonietta.gender);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
-
+kennan.speak = function(kennan) {
+  console.log(`Hello, my name is ${kennan}`)
+};
+console.log(kennan.speak());
```
Comment (GianBarGian) [inline assignments/objects.js:76]: Be careful, this is not returning what is expected. 
You declare your function with a parameter, but you are not passing any argument when you invoke it. 
You could want to use `this` here in this fashion:
`kennan.speak = function() {
  return 'Hello, my name is ${this.name}'
};`
Notice also you don't need to console.log inside the function, but to return the value that will be logged on line 76.

File: assignments/function-conversion.js [hunk 2]
```diff
@@ -4,27 +4,37 @@
...
-
+let myFunction = () => {
+    console.log("Function was invoked!")
+};
+myFunction();
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
+let anotherFunction = (param) => param;
```
Comment (GianBarGian) [inline assignments/function-conversion.js:15]: I believe you already know this, but in functions with just 1 param you could skip the parenthesis for added cleanness.

**Comment by GianBarGian** [comment]
Good job on the project Vincent, apart for a small mistake in your objects.js file everything is working fine.

Good job with your arrow functions and your for loops, they looks great. Keep up with the good work!

File: assignments/function-conversion.js (hunk 2 above)
Comment (iamcristos) [reply assignments/function-conversion.js:15]: Aiit thanks

File: assignments/objects.js (hunk 1 above)
Comment (iamcristos) [reply assignments/objects.js:76]: Ooops yea thanks

**Pull request merged**